### ISSUE TITLE
Where to start ?

### ISSUE THREAD

**Issue by DEEZZU** [git-abhinav/Sorting-visualization-in-R-and-C-#1]
@git-abhinav  Here is where i start ?

**Comment by git-abhinav** [comment]
@DEEZZU yes yes, have you seen the code? I got full marks in this assignment *remarks* checked by NG mam.

**Comment by DEEZZU** [comment]
> @DEEZZU yes yes, have you seen the code? I got full marks in this assignment _remarks_ checked by NG mam.

@git-abhinav if this is NG Mam's Assignment then I would like to move to another repo

**Comment by git-abhinav** [comment]
@DEEZZU Checkout the antivirus

**Comment by DEEZZU** [comment]
@git-abhinav  Rajan Sir wala assigment ? Yess absolutely!!

**Comment by git-abhinav** [comment]
@DEEZZU  yes, I made it with GUI
